views/layouts/main.blade.php

```diff
@@ -82,6 +82,10 @@
             window['{{ config('twill.js_namespace') }}'].twillLocalization = {!! json_encode($twillLocalization) !!};
             window['{{ config('twill.js_namespace') }}'].STORE = {};
             window['{{ config('twill.js_namespace') }}'].STORE.form = {};
+            window['{{ config('twill.js_namespace') }}'].STORE.config = {
+                publishDate24Hr: {{config('twill.publish_date_24h') ? 'true' : 'false'}},
```

Comment:
@ifox as it is in main.blade.php we do not have to set this up in the form or listing as this is globally available all the time.